heron/tools/ui/resources/static/js/plan-stats.js

```diff
@@ -60,6 +60,16 @@
         scaleTicks: [0, 500, 1000, 1500, 2000],
         format: function (d) { return d.toFixed(0); }
       },
+      {
+        name: 'BP',
```

Comment:
I intentionally named it BP in the heron/tools/ui/resources/static/js/plan-stats.js file as this name is too long and it gets cut while rendering, as shown below:
 
![image](https://cloud.githubusercontent.com/assets/1450850/25865571/28fe2c46-34a8-11e7-8a1c-82f36e29fbd4.png)

Instead of displaying Maximum Back Pressure it only displays Back Pressure. So I changed it to BP.